src/azure-cli/azure/cli/command_modules/iot/_help.py

```diff
@@ -908,3 +908,64 @@
     text: >
         az iot pnp repository update -r aaaabbbb11112222aaaabbbb1111222 -n updatedreponame
 """
+
+helps['iot central'] = """
+type: group
+short-summary: Manage IoT Central assets.
+"""
+
+helps['iot central app'] = """
+type: group
+short-summary: Manage IoT Central applications.
+"""
+
+helps['iot central app create'] = """
+type: command
+short-summary: Create an IoT Central application.
+long-summary: |
+    For an introduction to IoT Central, see https://docs.microsoft.com/azure/iot-central/.
+    The F1 Sku is no longer supported. Please use the ST2 Sku (default) for app creation.
+    For more pricing information, please visit https://azure.microsoft.com/pricing/details/iot-central/.
+examples:
+  - name: Create an IoT Central application in the standard pricing tier ST2, in the region of the resource group.
+    text: >
+        az iot central app create --resource-group MyResourceGroup --name my-app-resource --subdomain my-app-subdomain
+  - name: Create an IoT Central application with the standard pricing tier ST2 in the 'westus' region, with a custom display name, based on the iotc-pnp-preview template.
+    text: >
+        az iot central app create --resource-group MyResourceGroup --name my-app-resource-name --sku ST2 --location westus --subdomain my-app-subdomain --template iotc-pnp-preview --display-name 'My Custom Display Name'
+"""
+
+helps['iot central app delete'] = """
+type: command
+short-summary: Delete an IoT Central application.
+examples:
+  - name: Delete an IoT Central application. (autogenerated)
+    text: az iot central app delete --name MyIoTCentralApplication --resource-group MyResourceGroup
+    crafted: true
+"""
+
+helps['iot central app list'] = """
+type: command
+short-summary: List IoT Central applications.
+examples:
+  - name: List all IoT Central applications in a subscription.
```

Comment:
nit: List all 'the'

Reply:
Let's leave the text as is and not making any changes other than migration in this PR. We should make those types of change in a separate PR